[pytest11]
failures = pytest_failure_tracker.plugin
